Dockerfile

```diff
@@ -14,7 +14,20 @@ RUN apt-get update && \
         cd xml-doclet/; mvn install && \
         cd .. ; rm -rf xml-doclet && \
     sed 's/.ALL:ALL./(ALL) NOPASSWD:/' -i /etc/sudoers 
+RUN until apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv EF678DA6D5B1436D3972DFD317BE949418BE5D6B; do echo retrying; done && \
+    echo deb http://repos.demokracia.rulez.org/apt/debian/ master main >/etc/apt/sources.list.d/repos.demokracia.rulez.org.list && \
+    apt-get update
+ENTRYPOINT ["/usr/local/toolchain/tools/entrypoint"]
+CMD ["/bin/bash"]
+
+FROM base as build-stage0
```

Comment:
With multi-stage feature it's possible to build different images into the same Dockerfile. This practice is recommended in Good practices writing Dockerfiles.